### ISSUE TITLE
Add public, working examples.

### ISSUE THREAD

**Issue by danielballan** [AdvancedPhotonSource/xpcs-eigen#11]
The example should be exercised by CI so that we ensure they stay up to date.

**Comment by iamfaisalkhan** [comment]
Yes sure sounds good. I will be on parental leave starting from mid-August. Anytime after that should be good for me to work this.

**Comment by danielballan** [comment]
Oh, wonderful! Best wishes for your growing family!

**Comment by danielballan** [comment]
Now that #8 is closed, I think this is the next-highest priority as far as enabling collaborators is concerned.

**Comment by iamfaisalkhan** [comment]
I agree. We have something ready to go for this. In next couple of days, we will push few files along with the testing code to run against each of those as part of unit testing scripts. Hopefully those files can act as good examples.
